prdoc/pr_6321.prdoc

```diff
@@ -0,0 +1,32 @@
+title: pallet-utility: if_else
+
+doc:
+  - audience: Runtime Dev
+    description: This PR adds the `if_else` call to `pallet-utility`
+    enabling an error fallback when the main call is unsuccessful.
+
+crates:
+    - name: asset-hub-rococo-runtime
+      bump: patch
+    - name: asset-hub-westend-runtime
+      bump: patch
+    - name: bridge-hub-rococo-runtime
+      bump: patch
+    - name: bridge-hub-westend-runtime
+      bump: patch
+    - name: collectives-westend-runtime
+      bump: patch
+    - name: coretime-rococo-runtime
+      bump: patch
+    - name: coretime-westend-runtime
+      bump: patch
+    - name: people-rococo-runtime
+      bump: patch
+    - name: people-westend-runtime
+      bump: patch
+    - name: rococo-runtime
+      bump: patch
+    - name: westend-runtime
+      bump: patch
```

Comment:
Probably also major for all the runtimes, since it changes the RuntimeCall enum, but the CI should catch this.